workflow/Snakefile

```diff
@@ -16,6 +16,9 @@ report: "report/workflow.rst"
 container: "docker://condaforge/mambaforge"
 
 
+containerized: "quay.io/uncovar/uncovar:0.0.0"
```

Comment:
@johanneskoester Is this the correct format for quay.io?

Reply:
```suggestion
containerized: "quay.io/uncovar/uncovar#0.0.0"
```